core/src/test/scala/org/apache/spark/rdd/AsyncRDDActionsSuite.scala

```diff
@@ -27,7 +27,7 @@ import org.scalatest.BeforeAndAfterAll
 import org.scalatest.concurrent.Timeouts
 import org.scalatest.time.SpanSugar._
 
-import org.apache.spark.{LocalSparkContext, SparkContext, SparkException, SparkFunSuite}
+import org.apache.spark._
```

Comment:
The line needs to change regardless, because an import was added. Explicitly specifying 5 imported classes causes the line to exceed 100 characters, however.